disease_gene/disease_associates_gene/experiments/gen_model_benchmarking.py

```diff
@@ -0,0 +1,329 @@
+
+# coding: utf-8
+
+# # Generative Model Benchmarking
+
+# The goal here is to use the [data programing paradigm](https://arxiv.org/abs/1605.07723) to probabilistically label our training dataset for the disease associates gene realtionship. The label functions have already been generated and now it is time to train the generative model. This model captures important features such as agreements and disagreements between label functions; furthermore, this model can capture the dependency structure between label functions (i.e. correlations between label functions). More information can be found in this [blog post](https://hazyresearch.github.io/snorkel/blog/structure_learning.html) or in this [paper](https://arxiv.org/abs/1703.00854). The underlying hypothesis here is: **Modeling dependency structure between label functions has better performance compared to the conditionally independent model.**
+
+# ## Set up The Environment
+
+# The few blocks below sets up our python environment to perform the experiment.
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+from itertools import product
+import os
+import sys
+
+sys.path.append(os.path.abspath('../../../.'))
```

Comment:
Can we put `utils` in a directory with all your module code, such that python isn't searching every subdirectory of the root repo for code? For example, `modules/utils`.

Reply:
`../../../modules`